### PR TITLE
feat: cookie banner

### PR DESCRIPTION
**Pull request by Flosciante**

### CODE REVIEW

File: components/theme/app/AppLayout.vue
Comment (smarroufin) [inline components/theme/app/AppLayout.vue:21]: Really minor comment, but I feel like doing `inset-x-0` is more secure and simpler than `w-full left-0` 🙂

File: components/templates/CookieBanner.vue
Comment (smarroufin) [inline components/templates/CookieBanner.vue:54]: According to the docs it does not seem right to import them this way. Is there a reason?
https://github.com/Developmint/vue-if-bot#synchronous-import
https://github.com/EvodiaAut/vue-cookieconsent-component#usage

**Review by smarroufin — CHANGES_REQUESTED** [review]
Out of thoses two comments, it seems fine and works well. Probably wait for a second review, from someone actually knowing cookies 😅 .

File: components/theme/app/AppLayout.vue
Comment (Flosciante) [reply components/theme/app/AppLayout.vue:21]: 👍

File: components/templates/CookieBanner.vue
Comment (manniL) [reply components/templates/CookieBanner.vue:54]: I think I have to update the docs 🙈 
`vue-if-bot` import seems alright 👍 
https://github.com/manniL/lichter.io/blob/31cded247d5c2cad2bb6f29be5355e46bec9ffdb/layouts/default.vue#L18

**Pull request merged**
